### PR TITLE
Black

### PR DESCRIPTION
**Pull request by graingert**
### Contributor Checklist:

* [ ] I have updated the release notes at `docs/source/NEWS.rst`
* [x] I have updated the automated tests.
* [ ] All tests pass on your local dev environment. See `CONTRIBUTING.rst`.

### CODE REVIEW

**Comment by graingert** [comment]
@adiroiban to review this I'd recommend checking out 18fc3f2 then running

```sh
pipx run pre-commit run --all-files
pipx run pre-commit run --all-files  # a second time so that pyupgrade can run on the new blackened source
```

**Comment by graingert** [comment]
black touches everything and so I think it's best to ignore the codecov/patch failures here

File: .git-blame-ignore-revs [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+# You can configure your checkout to use it by default by running:
+#
+#   git config blame.ignoreRevsFile .git-blame-ignore-revs
+#
+# Or pass it to the blame command manually:
+#
+#   git blame --ignore-revs-file .git-blame-ignore-revs ...
+#
+# Note that these features require Git 2.23 (released August 2019).
+355c65a31f107bca5d340f00a234f7909fb51fae
```
Comment (adiroiban) [inline .git-blame-ignore-revs:11]: Is this correct.
'fade to black' commit is 86ef1de2a30fe3f5a59f371b9f737390781a1090

**Review by adiroiban — APPROVED** [review]
Thanks. Let's do this.

Please see my comment to double check that we ignore the right commit.

Thanks!

File: .git-blame-ignore-revs (hunk 1 above)
Comment (graingert) [reply .git-blame-ignore-revs:11]: Oh woops

**Review by adiroiban — APPROVED** [review]
Thanks

**Pull request merged**